src/integrations/thumbnail.ts

```diff
@@ -126,28 +124,22 @@ export const get: APIRoute = async ({ url }) => {
 		}	
 	} catch (error) {
 		// If everything breaks, use a default image
-		console.error(error)
+		// console.error(error)
 		const image = await fetch('https://cloud-i203j2e6a-hack-club-bot.vercel.app/1confused_dinosaur.png')
 		thumbnail = {
 			kind: 'png',
 			data: Buffer.from(await image.arrayBuffer()).toString('base64')
 		}
 	}
 
-	return new Response(JSON.stringify(thumbnail), {
-		status: 200,
-		headers: {
-			'Access-Control-Allow-Credentials': 'true',
-			'Access-Control-Allow-Origin': '*',
-			'Access-Control-Allow-Methods': 'GET,OPTIONS',
-			'Access-Control-Allow-Headers': '*',
-			'Cache-Control': 'max-age=86400'
-		}
-	})
+	// write/overwrite image json to public folder
+	let imgFilePath = path.resolve(__dirname, `../../public/${name}.json`);
+	fs.writeFileSync(imgFilePath, JSON.stringify(thumbnail), { encoding: "utf-8" });
```

Comment:
Please ensure this operation succeeds, and if not have it at least log something.